bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,22 +24,12 @@
         {% if not hide_nav_download_button %}
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
-          {% if LANG in ['en-US', 'de', 'fr'] %}
-            {% if LANG == 'de' %}
-              {% set fxa_button_text = 'Firefox-Konto erstellen' %}
-              {% set fxa_link_text = 'Das sind deine Vorteile' %}
-            {% elif LANG == 'fr' %}
-              {% set fxa_button_text = 'Créez un compte Firefox' %}
-              {% set fxa_link_text = 'Découvrez les avantages' %}
-            {% else %}
-              {% set fxa_button_text = 'Get a Firefox Account' %}
-              {% set fxa_link_text = 'Check out the Benefits' %}
-            {% endif %}
+          {% if LANG in ['en-US', 'en-CA', 'en-GB', 'en-ZA', 'de', 'fr'] %}
             <div class="c-navigation-fxa-cta-container">
-              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta">
-                {{ fxa_button_text }}
+              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta" data-alt-href="{{ url('firefox.accounts') }}">
```

Comment:
`mzp-t-download` no longer exists now that bedrock is on Protocol 5.0, should be `mzp-t-product`.